src/server.c

```diff
@@ -3430,6 +3430,11 @@ void rejectCommandFormat(client *c, const char *fmt, ...) {
     sdsfree(s);
 }
 
+static int command_can_handle_movablekeys(struct redisCommand *cmd) {
```

Comment:
please rename to `cmdHasMovableKeys` (redis is using camelCase for functions).
also, please add a top comment explaining what the return value means,
maybe something like:

```
A command that uses keys but has no pre-determined key position arguments.
```

Reply:
fixed